schema-validation-lib/dev/user.clj

```diff
@@ -0,0 +1,14 @@
+(ns user
+  (:require [clojure.pprint :refer (pprint pp)]
```

Comment:
Looks like this was copied from somewhere that had not been updated yet to match our namespace conventions adopted from here: https://stuartsierra.com/2016/clojure-how-to-ns.html